packages/ui-text/src/Text/styles.js

```diff
@@ -87,11 +87,11 @@ const generateStyle = (componentTheme, props, state) => {
 
   const letterSpacingVariants = {
     normal: componentTheme.letterSpacingNormal,
-    condensed: componentTheme.letterSpacingNormal,
-    expanded: componentTheme.letterSpacingNormal
+    condensed: componentTheme.letterSpacingCondensed,
+    expanded: componentTheme.letterSpacingExpanded
```

Comment:
This is an additional bugfix I found, all letter spacing values were set to normal. Will need to accept this as a new baseline in VRT too.